components/AdminLayout.js

```diff
@@ -52,38 +71,41 @@ export default function AdminLayout({ children, host, siteUrl }) {
         <script async src="https://apis.google.com/js/client:platform.js" />
       </Head>
       <main className="container">
-        {!session && !cypressTesting && !skipAuth && (
-          <section tw="bg-gray-200 text-gray-900 relative">
-            <div tw="min-h-screen bg-right-top bg-cover flex">
-              <div tw="relative container mx-auto p-4 flex items-center z-10">
-                <div>
-                  <div tw="content float-left py-4 px-5 my-5">
-                    <div tw="mb-3 text-2xl md:text-4xl">TinyCMS</div>
-                    <div tw="leading-normal hidden sm:block">
-                      You must be signed in to use these tools.
+        {(!session && !cypressTesting && !skipAuth) ||
+          (!isAllowedToAccess && (
```

Comment:
I think we should have an explicit message on this page for when someone attempts to login with an invalid domain.

Reply:
Okay, I'll add this today.